### DISCUSSION TITLE
About version of ericlagergren/decimal package

### DISCUSSION THREAD

**Discussion by ParkJeongwoong** [aarondl/sqlboiler#1429] [Q&A] [answered]
In the README, it says [using the 20181231230500 version of the ericlagergren/decimal package ](https://github.com/volatiletech/sqlboiler#:~:text=decimal%20library%20showing%20errors%20like%3A%20pq%3A%20encode%3A%20unknown%20type%20types.NullDecimal%20is%20a%20result%20of%20a%20too%2Dnew%20and%20broken%20version%20of%20the%20github.com/ericlargergren/decimal%20package%2C%20use%20the%20following%20version%20in%20your%20go.mod%3A%20github.com/ericlagergren/decimal%20v0.0.0%2D20181231230500%2D73749d4874d5)to avoid certain errors.
However, starting from sqlboiler v4.14.0, the library switched to github.com/ericlagergren/decimal v0.0.0-20211103172832-aca2edc11f73. Later, in v4.15.0, the version was downgraded to v0.0.0-20190420051523-6335edbaa640 (from this [PR](https://github.com/volatiletech/sqlboiler/commit/bc59c158590800f7810cce241a12d572e898014f)).

Does the recommendation in the README to avoid versions newer than 20181231230500 still apply to the latest versions?
Or does README needed to be updated to use 20190420051523 version of ericlagergren/decimal?

**Comment by stephenafamo** [comment, marked as answer]
I believe it no longer applies, the README is likely outdated now.
